src/dbSta/src/dbReadVerilog.cc

```diff
@@ -139,12 +144,29 @@ class Verilog2db
   Verilog2db(Network* verilog_network, dbDatabase* db, Logger* logger);
   void makeBlock();
   void makeDbNetlist();
+  static string makePinName(std::string& name);
```

Comment:
done.
I wanted to have a generic make pin name so that wherever a pin name is made it can be sanitized (there is some ambiguity elsewhere in the code about pin name construction and the vain hope was to have a nice globally visible method to construct a clean pin name). But no problem made private and likewise the input pin name is now const.